package.json

```diff
@@ -71,8 +71,10 @@
   },
   "dependencies": {
     "find-up": "^5.0.0",
+    "load-tsconfig": "^0.2.5",
```

Comment:
Where is this used?

Reply:
ts-up is complaining about it

Reply:
ts-up was added to bundle when we decide to build without nextjs